modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/fragment/renderer/dynamic_field/js/CPInstanceChangeHandler.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import * as Events from 'commerce-frontend-js/utilities/eventsDefinitions';
+
+const deep_value = (obj, path, defaultValue) => {
```

Comment:
just check the function name pattern I'm not sure about the _ inside the function name